### DISCUSSION TITLE
What happened to Gamma and Kappa teams?

### DISCUSSION THREAD

**Discussion by thevibingcat** [redeclipse/base#1359] [Help and Support] [answered]
in red eclipse 1, there were 4 team maps, that would feature 2 more teams, Gamma (green) and Kappa (yellow). you can still play 4 team maps on some servers in red eclipse 2, but the logos are just missing textures. if you go into an editing server, and look at the textures, some textures are yellow and green colored. I wish to know why Gamma and Kappa were removed in Red eclipse 2

**Comment by Jigoku** [comment, marked as answer]
Hi, the _multi_ mutator has been removed since 2.0. I believe the main reason for its removal was that matches would generally be chaotic, and it was very hard to balance maps when taking 4 teams into consideration.
